pubkeyConverter/bech32PubkeyConverter.go

```diff
@@ -0,0 +1,137 @@
+package scenexpressioninterpreter
```

Comment:
Would be nice to reuse the entire NewBech32PubkeyConverter from mx-chain-core-go.

I see we are already importing the package. All we need is to find a logger that does nothing.